akka-actor/src/main/scala/akka/actor/CoordinatedShutdown.scala

```diff
@@ -344,6 +344,7 @@ final class CoordinatedShutdown private[akka] (
       knownPhases(phase),
       s"Unknown phase [$phase], known phases [$knownPhases]. " +
         "All phases (along with their optional dependencies) must be defined in configuration")
+    require(!taskName.isEmpty)
```

Comment:
Nitpick: `require(taskName.nonEmpty)` ;).

Reply:
should also include error message